lib/motions/general-motions.coffee

```diff
@@ -201,7 +201,18 @@ class MoveToNextWord extends Motion
   wordRegex: null
   operatesInclusively: false
 
+  wordRegExp: (cursor, {includeNonWordCharacters}={}) ->
```

Comment:
Is this method ever called with a second argument? I only see the one call on line 215.